build/Toolset.proj

```diff
@@ -4,6 +4,6 @@
     <TargetFramework>net462</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="RoslynTools.Microsoft.RepoToolset" Version="$(RoslynToolsMicrosoftRepoToolsetVersion)" />
+    <PackageReference Include="RoslynTools.RepoToolset" Version="$(RoslynToolsRepoToolsetVersion)" />
```

Comment:
You can also delete nuget.config from the root directory and instead add 

<RestoreSources>[https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json](https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json)</RestoreSources>

here

Reply:
The other feeds are included by RepoToolset automatically.

---
In reply to: [157092889](https://github.com/dotnet/roslyn-sdk/pull/27#discussion_r157092889) [](ancestors = 157092889)

Reply:
xliff also gets pulled in?